models/user_repository.go

```diff
@@ -0,0 +1,37 @@
+package models
```

Comment:
@aslakknutsen There are two identity repositories, the one inside account package and another one inside application package with incompatible method signatures. So, I have used the one inside application package.